tests/src/Kernel/fixtures/rendering.yml

```diff
@@ -2670,12 +2670,14 @@
           text: "Culture | Innovation"
         -
           icon: "calendar"
+          size: "s"
```

Comment:
```suggestion
          size: "xs"
```

Reply:
No need for this change since this is testing the pattern.